Gruntfile.js

```diff
@@ -148,7 +114,7 @@ module.exports = function (grunt) {
             },
             all: {
                 files: {
-                    'build/temp/dash.no-externals.debug.js': ['src/All.js']
+                    'build/temp/dash.all.min.js': ['src/All.js']
```

Comment:
This should be something like `build/temp/dash.all.debug.js`

Then we would have an uglify step that would convert `dash.all.debug.js` into `dash.all.min.js`. The `dist` copy will need to be upgraded accordingly.